packages/react-components/src/components/TogetherModeOverlay.tsx

```diff
@@ -182,6 +182,9 @@ export const TogetherModeOverlay = memo(
                 }}
                 onMouseEnter={() => setHoveredParticipantID(participantStatus.id)}
                 onMouseLeave={() => setHoveredParticipantID('')}
+                onKeyUp={(e) => handleKeyUp(e, participantStatus.id)}
+                onBlur={() => setTabbedParticipantID('')}
+                tabIndex={index}
```

Comment:
This should always be 0. by setting a non 0 value you are changing the order of the tabIndex of the page. by having the value here as 0 you are making sure each item is part of the natural tab order of the page and interactive. it is not recommended to give something a tabIndex > 0. the natural order will take the order from top -> bottom, left -> right (unless rtl is on then its the other way)

as an aide: -1 makes an item non navigable and should not be used unless that is the intent.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex